apps/mobile/src/components/OfferInfoPreview.tsx

```diff
@@ -52,19 +52,8 @@ function OfferInfoPreview({
   )
 
   const offerAmount = useMemo(() => {
-    if (offer.publicPart.currency === 'CZK')
-      return `${bigNumberToString(offer.publicPart.amountTopLimit)} ${t(
-        'common.czkSymbol'
-      )}`
-    if (offer.publicPart.currency === 'EUR')
-      return `${t('common.eurSymbol')}${bigNumberToString(
-        offer.publicPart.amountTopLimit
-      )}`
-    if (offer.publicPart.currency === 'USD')
-      return `${t('common.dollarSymbol')}${bigNumberToString(
-        offer.publicPart.amountTopLimit
-      )}`
-  }, [offer.publicPart.amountTopLimit, offer.publicPart.currency, t])
+    return `${bigNumberToString(offer.publicPart.amountTopLimit)} ${offer.publicPart.currency}`
```

Comment:
Since the app is not displaying currency symbols in this code review, it is ok always display currency after the amount right?

Reply:
Yes, that was my idea for this change.